proposal.md

```diff
@@ -14,28 +15,38 @@ Summary
 
 
 > "Four shalt thou not count, neither shalt thou count two, excepting that thou
-then proceedeth to three."
+then proceed to three."
 
-Python 3 has been around for now more than eight years, and much of the Python
+Python 3 has been around for more than eight years, and much of the Python
 ecosystem is now available both on Python 2 and Python 3, often using a single
 code base. There comes a time in the life of such libraries when the authors
-start to consider stopping support for Python 2 in a new major version.
-
-While the above statement seem to be relatively simple, it was (or is,
-depending on the merge-state of various patches) difficult to achieve without
-wreaking havoc for Python 2 users, as up until recently is was not possible to
-tag a release as Python 3 only.
-
-Like any maintainer of a widely used library, we care deeply about ensuring
-that users still on Python 2 can easily install older versions, or a parallel
-"Long Term Support" variant, which still works for them. Doing so without
-having users to explicitly pin dependencies in all their projects was not easy
-easy task. Thus we ventured into the rabbit-hole of Packaging to make both
-users and maintainers life easier.
-
-We'll narrate our tale through the amending of PEPs, our journeys with the
-knights of setuptools, contribution to the Castle of pypa/Warehouse, fights with
-the dragons of Pip, and errands in the unittest-less land of PyPI legacy.
+consider stopping support for Python 2 in a new major version.
+
+While it may seem simple to cease support for Python 2 – if you did not want
+wreak havoc for users who wish to stay on Python 2 – you just made your life
+much more difficult. Put simply, up until recently it was not possible to tag
+a release as Python 3 only, today it is possible. 
```

Comment:
I'm working on the assumption that this will be true by PyCon, I think we have a stronger case if we can make this claim, and what we can say in a preamble as an extra note is just that this the text presumes that some of these fixes will go through by the time of the presentation, and that we will modify the content of the talk accordingly to describe the – hopefully, few – problems that still need to be faced if those problems still exist.